src/mmw/js/src/projects/controllers.js

```diff
@@ -4,6 +4,10 @@ var App = require('../app'),
     views = require('./views');
 
 var ProjectsController = {
+    projectsCleanUp: function() {
```

Comment:
[`projectsCleanup` already exists below](https://github.com/WikiWatershed/model-my-watershed/blob/54d7d6244e537fb7c051dbb2ae2d2189ee20e5db/src/mmw/js/src/projects/controllers.js#L15-L17), consider adding the line

```javascript
App.aoiCensus = null;
```

to it instead.